game/src/test/scala/io/aigar/controller/AigarStackSpec.scala

```diff
@@ -0,0 +1,38 @@
+import io.aigar.controller._
+import io.aigar.controller.response.{ErrorResponse}
+
+import org.json4s.{DefaultFormats, Formats}
+import org.json4s.jackson.JsonMethods._
+
+import org.scalatra.test.specs2._
+import org.specs2.matcher._
+
+object ConcreteStack extends AigarStack {
+  get("/:id") {
+    halt(params("id").toInt)
+  }
+}
+
+class AigarStackSpec extends MutableScalatraSpec {
+  implicit val jsonFormats: Formats = DefaultFormats
+
+  addServlet(ConcreteStack, "/*")
+
+  "errors" should {
+    "404 should return a generic error" in {
```

Comment:
If we were to do the "generic" error trapping in my comment above, what if we would replace this by something along the lines of `404 should be trapped as an ErrorResponse`?
Would maybe make sense to also test that `200` is not being trapped?